products/statement-generator/public/locales/en/translation.json

```diff
@@ -23,7 +23,7 @@
     "sectionParagraph2": "Expunge Assist is a non-profit organization under Hack for LA. We aim to help people with criminal records accomplish record clearance, expungement, or a reduction to their sentence.\n\nAs of now, this service is only available to people who have criminal records in California.",
     "sectionTitle3": "Making letter writing less intimidating",
     "sectionParagraph3": "This is a tool for applicants who are writing a letter to a judge that addresses their progress and motivations for seeking expungement.",
-    "sectionParagraph3BulletPoint1": "Targeted question prompts guide the writer",
```

Comment:
Typically I would have this separated into a separate issue because it's unrelated to issue #1079 , however it's so small that I'll let it slide.